src/timeline-list/useTimelinePages.ts

```diff
@@ -7,18 +7,22 @@ import debounce from 'lodash/debounce';
 import constants from '../commons/constants';
 import {generateDay} from '../dateutils';
 
-const PAGES_COUNT = 100;
+export const PAGES_COUNT = 100;
 export const NEAR_EDGE_THRESHOLD = 10;
-export const INITIAL_PAGE = Math.floor(PAGES_COUNT / 2);
-
 
 interface UseTimelinePagesProps {
   date: string;
   listRef: RefObject<any>;
+  numberOfDays: number;
 }
 
-const UseTimelinePages = ({date, listRef}: UseTimelinePagesProps) => {
-  const pagesRef = useRef(times(PAGES_COUNT, i => generateDay(date, i - Math.floor(PAGES_COUNT / 2))));
+const UseTimelinePages = ({date, listRef, numberOfDays}: UseTimelinePagesProps) => {
+  const initialPage = Math.ceil(PAGES_COUNT / 2);
```

Comment:
What's the purpose of moving `initialPage` here? 
I see you now calculate it in other files as well..